### ISSUE TITLE
alternatively web

### ISSUE THREAD

**Issue by icacaretoomuch** [deltachat/deltachat-ios#4]
For non-android users what options are there for interacting with android users of DeltaChat?  ... if not a DeltaChat app

web based thinggy?

**Comment by r10s** [comment]
Non-Delta-Chat-Users can interact with Delta-Chat-Users with _any_ e-mail client.

Of course, an explicit Delta Chat client will make things far more comfortable :)
